### DISCUSSION TITLE
button mini-scripts

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#6135] [Engine Core]
I think that buttons and alike nodes should have the ability to trigger something in a parent or grandparent node e.g
`get_parent().hide` for hiding their parent instead of having to connect the signal to the nearest script 4 parents above it or something

**Comment by dalexeev** [comment]
It's already possible, but it's kind of a secret feature.

![](https://user-images.githubusercontent.com/47700418/213861253-b0fd2409-24bd-4601-8b77-3d3679d67baf.png)
